mitiq/zne/scaling/tests/test_folding.py

```diff
@@ -1831,12 +1835,12 @@ def test_apply_fold_mask():
     assert _equal(folded, correct)
 
     folded = _apply_fold_mask(circ, [0, 3, 0, 0, 0, 0])
-    correct = Circuit([ops.H.on(qreg[1])] * 6) + circ
-    assert _equal(folded, correct)
+    correct = Circuit(*[ops.H.on(qreg[1])] * 6) + circ
```

Comment:
Is the unpacking `*` needed here? If so, what for?

Reply:
It's not actually. Removed in [2fdc9ce](https://github.com/unitaryfund/mitiq/pull/1379/commits/2fdc9cec0fe904d046c3ce476fd8f2072aa5eda2)